### PR TITLE
Move Customizing the installation configuration into a separate page

### PR DESCRIPTION
**Pull request by shamsher31**
To fix https://github.com/istio/istio.io/issues/9642

### CODE REVIEW

**Comment by ericvn** [comment]
Waiting on approval from @istio/wg-environments-maintainers.

File: content/en/docs/setup/install/istioctl/index.md
Comment (howardjohn) [inline content/en/docs/setup/install/istioctl/index.md:306]: can we have a link to the other doc here? otherwise most will not find it

Comment (shamsher31) [reply content/en/docs/setup/install/istioctl/index.md:306]: Added. Thanks.

**Comment by shamsher31** [comment]
@ericvn PTAL.

File: content/en/docs/setup/additional-setup/customize-installation/index.md
Comment (ericvn) [inline content/en/docs/setup/additional-setup/customize-installation/index.md:5]: I don't think we need an alias here as this is a new page (extracted from a page that still exists).

File: content/en/docs/setup/install/istioctl/index.md
Comment (ericvn) [inline content/en/docs/setup/install/istioctl/index.md:306]: See [Customizing the installation configuration](/docs/setup/additional-setup/customize-installation/) for additional information on customizing the install.

**Review by ericvn — APPROVED** [review]

**Pull request merged**
